source/_components/device_tracker.tomato.markdown

```diff
@@ -23,20 +23,56 @@ To use this device tracker in your installation, add the following to your `conf
 # Example configuration.yaml entry
 device_tracker:
   - platform: tomato
-    host: YOUR_ROUTER_IP_ADDRESS
+    host: YOUR_ROUTER_IP_ADDRESS_OR_HOSTNAME
+    port: YOUR_ROUTER_PORT
```

Comment:
Optional configuration variables should not be in the examples.
We are trying to keep the examples as minimal as possible.

Reply:
Sounds good!

Reply:
It might be a good thing to call out that ^^ in the standards documentation since I don't think it mentions optional variables not being part of the examples.